scripts/check_compliance.py

```diff
@@ -401,6 +401,7 @@ def check_no_undef_outside_kconfig(self, kconf):
     "CONFIG_REG1",
     "CONFIG_REG2",
     "CONFIG_SEL",
+    "CONFIG_SHIFT", #From STM32MP1 Cube (ext/hal/), file stm32mp1xx_ll_rcc.h
```

Comment:
Nit: Please put a space after `#` (and `/*`, and `//`) in the future. It's a much more common style.

Reply:
Will remember this for next time, thank you !